### PR TITLE
4600 - Tabs (Split and Drag)

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR adds the ability to split tabs.

In this approach, the tabs will
- Tabs can be transferred via drag and drop
- Tabs can be splitted via dragging the tab below and releasing it.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/4600

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/tabs-multi/example-drag-split.html

**Transfer Tabs from Tab Group to another Tab Group**
- Drag the `Contracts` tab to the another tab Group on the right side.
- The Tab Group where the `Contracts` tab came from will be hidden.

**Tabs Splitting**
- From the example above, drag any of the tabs below and then release it. (In example. `Attachments`)
- The dragged tab will then be split from that Tab Group and will be transferred to another Tab Group.

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

File: src/components/tabs-multi/multi-tabs.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ const COMPONENT_NAME = 'multitabs';
 
 // Default Settings for MultiTabs
 const MULTITABS_DEFAULTS = {
-  tabContainers: []
+  tabContainers: [],
+  sortableTabs: false
```
Comment (tmcconechy) [inline src/components/tabs-multi/multi-tabs.js:12]: In other places we uses "sortable" as a setting.

**Review by tmcconechy — COMMENTED** [review]
Looks like a great start but in terms of behavior

1) When i drag from the right to left the tab "snaps back" and then loads and seems to take a while. I would expect it not to snap back
2) When i drag the one on the left over to the right so there is one set then how can i split tabs. I think this is the use case on the issue where they start with no splits and then drag a tab ON another tab to make it split.

**Comment by tjamesallen15** [comment]
Will work on this one. Thanks for reviewing @tmcconechy

**Comment by tjamesallen15** [comment]
To split the tabs, drag one tab below and then release it. 
015A8D90-D9E2-49DD-B4C7-09522C78531E_1_105_c

**Comment by tmcconechy** [comment]
I think this is looking really good. The only thing i would do is instead of grey when you drop it down the first time on the shadow: try azure 01 #E6F1FD

Otherwise i think this works perfectly.

**Review by tmcconechy — APPROVED** [review]

File: src/components/tabs/_tabs.scss [hunk 2]
```diff
@@ -932,6 +932,27 @@ html.is-firefox {
   }
 }
 
+.tab-panel-container {
+  .tab-overlay {
+    position: fixed;
```
Comment (ericangeles) [inline src/components/tabs/_tabs.scss:937]: @tmcconechy it looks like that the stylelint doesn't work anymore? I remember that we have lint for alphabetical order in CSS/SCSS.

**Review by ericangeles — APPROVED** [review]

File: src/components/tabs/_tabs.scss (hunk 2 above)
Comment (ericangeles) [reply src/components/tabs/_tabs.scss:937]: @tmcconechy I like the ordering by group type. I'll keep this in mind. Thanks! :) FYI @tjamesallen15 @yohannahbautista

**Pull request merged**